src/core/pattern.js

```diff
@@ -198,7 +198,7 @@ Shadings.RadialAxial = (function RadialAxialClosure() {
         r1 = coordsArr[5];
         type = 'radial';
       } else {
-        error('getPattern type unknown: ' + shadingType);
+        throw new Error(`getPattern type unknown: ${shadingType}`);
```

Comment:
This case shall never happen -- getIR() is usually called after parsing. Changing that to `unreachable()` call -- this call can be safely remove (it's similar to `assert()`)